.github/workflows/verify_microsite.yml

```diff
@@ -35,6 +35,9 @@ jobs:
         with:
           node-version: 18.x
 
+      - name: enable corepack
+        run: corepack enable
```

Comment:
Super nit but I find the name on these to be a bit redundant, thinking `- run: corepack enable` is enough